app/templates/application.hbs

```diff
@@ -6,6 +6,7 @@
     {{/if}}
     <div id="main-container"
          class="ui container main-container {{unless device.isInternetExplorer 'flexed-container' ''}}">
+      {{filter-option}}
```

Comment:
We want this **only** on the home page. If you put it here, it'll be displayed on all pages.